bloptools/de_opt_utils.py

```diff
@@ -123,9 +123,16 @@ def calc_velocity(motors, dists, velocity_limits, max_velocity=None, min_velocit
 
 def _run_flyers(flyers):
     uid_list = []
-    for flyer in flyers:
-        uid = (yield from bp.fly([flyer]))
-        uid_list.append(uid)
+    if isinstance(flyers[0], list):
+        for flyer in flyers:
+            # pass flyer into bp.fly since flyer is already a list of flyers
+            uids = (yield from bp.fly(flyer))
+            uid_list.append(uids)
+    else:
+        for flyer in flyers:
+            # pass [flyer] into bp.fly since flyer isn't a list here
+            uid = (yield from bp.fly([flyer]))
+            uid_list.append(uid)
```

Comment:
Instead of repeating the code, can we redefine `flyers = flyers[0]` in this case? Also, we should check that the flyers list is not empty (which is potentially possible input, but makes no sense for simulations).